src/dc_receive_imf.rs

```diff
@@ -238,6 +237,19 @@ pub fn dc_receive_imf(
         );
     }
 
+    if mime_parser.user_avatar != AvatarAction::None {
+        match contact::set_profile_image(&context, from_id, mime_parser.user_avatar) {
+            Ok(()) => {
+                context.call_cb(Event::ChatModified(chat_id));
+                true
+            }
+            Err(err) => {
+                warn!(context, "reveive_imf cannot update profile image: {}", err);
+                false
```

Comment:
these true/false values are not used, are they?

Reply:
no, but if i leave them out, the match-arms have different returns.

is `()` the rust-way to go here?